clusterBuildStrategy/buildah/README.md

```diff
@@ -0,0 +1,103 @@
+# Buildah ClusterBuildStrategy
+The `buildah` ClusterBuildStrategy uses [buildah](https://github.com/containers/buildah) to build and push a container image, out of a `Dockerfile`. The `Dockerfile` should be specified using a parameter in the `Build` resource.
+
+## Getting Started with Buildah ClusterBuildStrategy
+
+### Pre-requisites:
+- The `Builds for Red Hat OpenShift` Operator is available in the OpenShift cluster.
+- The `buildah` clusterBuildStrategy is already installed. This could be checked by following command:
+```
+$ oc get cbs | grep buildah
+```
+If the buildStrategy is not available, [Install ClusterBuildStrategies](../../README.md) first.
+
+### Steps:
+
+1. Create a builds object. 
+- `source` refers to the location where the source code is.
+- `strategy` references the BuildStrategy to use to build the container.
+- `paramValues` is a list of key/value that could be used to set strategy parameters.
+    - Here, we pass the `dockerfile` parameter to specify the Dockerfile's location required to build the output image.
+- `output` refers to the location where the built image would be pushed.
+    - Here, it is expected to be pushed to the OpenShift cluster’s internal registry.
+    - `buildah-example` is the name of the current project.
+```
+$ cat <<EOF | oc create -f -
```

Comment:
I think we want the following:

1. Use `oc apply` instead of `create` - apply adds important metadata to Kubernetes objects.
2. I don't think we need the leading `cat` command?

```suggestion
$ oc apply -f - <<EOF
```